NEWS.md

```diff
@@ -4,6 +4,25 @@
 
 - Website now has button/links to Slack channel and GitHub Issues. (#262)
 
+## Documentation
```

Comment:
do we want to add new version name? and info about iRECIST vignette?

Reply:
The new version name is added in a separate PR which prepares the package for submission.

A news item for the iRECIST vignette should be added in the iRECIST branch.